### PR TITLE
"New loss" tutorial, docstring fixes.

### PR DESCRIPTION
**Pull request by AstitvaAggarwal**
## Checklist

- [ ] Appropriate tests were added
- [ ] Any code changes were done in a way that does not break public API
- [ ] All documentation related to code changes were updated
- [ ] The new code follows the
  [contributor guidelines](https://github.com/SciML/.github/blob/master/CONTRIBUTING.md), in particular the [SciML Style Guide](https://github.com/SciML/SciMLStyle) and
  [COLPRAC](https://github.com/SciML/COLPRAC).
- [ ] Any new documentation only uses public API
  
## Additional context
BPINNs improvement is evident in tests, adding in tutorial made it too long imo. Will add the same for PINO in a seperate PR.

### CODE REVIEW

**Comment by AstitvaAggarwal** [comment]
@ChrisRackauckas, lmk what you think. Hope this can help with the presentation. (the current improvement has been shown for PINNs and also highly cited BPINNs papers. It completes the BPINN model formulation). And I think Inverse problems are the main thing PINNs are currently used for.

File: src/ode_solve.jl [hunk 1]
```diff
@@ -49,7 +49,7 @@ standard `ODEProblem`.
 * `strategy`: The training strategy used to choose the points for the evaluations.
               Default of `nothing` means that `QuadratureTraining` with QuadGK is used if no
               `dt` is given, and `GridTraining` is used with `dt` if given.
-* `estim_collocate`: A boolean value to indicate whether to use the new loss function or not. This is only relevant for ODE parameter estimation.
+* `estim_collocate`: A boolean value to indicate whether to use the Data Quadrature loss function or not. This is only relevant for ODE parameter estimation.
```
Comment (ChrisRackauckas) [inline src/ode_solve.jl:52]: It's not necessarily tied to inverse problems though? One use case for it is using data from a forward solve in order to get a faster/more stable fit.

Comment (AstitvaAggarwal) [reply src/ode_solve.jl:52]: Oh yeah, ill change that up quick

Comment (AstitvaAggarwal) [reply src/ode_solve.jl:52]: Although conceptually true, as of now ive implemented it such that it works primarily for inverse problems. Maybe I change it up in the next PR (together with adding this loss for PINO)?

Comment (ChrisRackauckas) [reply src/ode_solve.jl:52]: Okay then let's merge and then extend. Though your example is exactly the use case of first solving with an ODE solver and then building a surrogate (and using the quadrature points to improve convergence), so the tutorial is a good highlight of this anyways.

**Comment by AstitvaAggarwal** [comment]
@ChrisRackauckas im not sure if random initializing is a good idea in tests that have some kind of fixed tolerances, but im including it anyways. Tests should pass now.

**Comment by AstitvaAggarwal** [comment]
@ChrisRackauckas tests pass now, should be fine to merge.

**Comment by ChrisRackauckas** [comment]
https://buildkite.com/julialang/neuralpde-dot-jl/builds/3072#0197096b-97d5-48eb-9753-89779f8377eb/767-2762

**Comment by AstitvaAggarwal** [comment]
@ChrisRackauckas that error is not related to my tutorial. https://buildkite.com/julialang/neuralpde-dot-jl/builds/3072#0197096b-97d5-48eb-9753-89779f8377eb/767-1922

File: docs/src/tutorials/ode_parameter_estimation.md [hunk 2]
```diff
@@ -54,7 +54,7 @@ Next we define the optimizer and [`NNODE`](@ref) which is then plugged into the
 ```@example param_estim_lv
 opt = LBFGS(linesearch = BackTracking())
 alg = NNODE(chain, opt, ps; strategy = WeightedIntervalTraining([0.7, 0.2, 0.1], 500),
-    param_estim = true, additional_loss)
+    param_estim = true, additional_loss = additional_loss)
```
[the remark this answers is not in the document]
Comment (AstitvaAggarwal) [reply docs/src/tutorials/ode_parameter_estimation.md:57]: Actually it dint, it was working fine. The issue was somewhere else.
